.github/workflows/main.yml

```diff
@@ -0,0 +1,52 @@
+name: Pull Request
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on pull request events for the develop and master branches
+  pull_request:
+    branches: [ develop, master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "test"
+  test:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a set of commands using the runners shell
+      - name: Build and Test project
+        run: |
+          echo Build and Test project
+          ./mvnw clean verify
+
+      - name: Coverage Information
+        id: coverage_info
+        run: |
+          per_package_info=$(awk -F "," '/three-hundred-sixty/ { missed[$2] += $4 } { covered[$2] += $5  } END { for(x in missed)printf ("%s %s%%\n", x, (covered[x]/(covered[x]+missed[x]))*100) }' target/site/jacoco/jacoco.csv)
+
+          per_package_info="${per_package_info//'%'/'%25'}"
+          per_package_info="${per_package_info//$'\n'/'%0A'}"
+          per_package_info="${per_package_info//$'\r'/'%0D'}"
```

Comment:
This is kind of a hack. I don't like having to do this.

It's based on this answer in the GH Community: https://github.community/t/set-output-truncates-multiline-strings/16852

Reply:
Thanks for the links..

Just curiosity, according this thread this escaping issue happen in bash but not in javascript actions

> Escaping of special characters already happened for javascript actions but does not for bash scripts. There is a plan to update documentation.
> (andreasplesch, Apr '20)